pkg/e2e/verify/routes.go

```diff
@@ -33,54 +27,48 @@ func init() {
 	alert.RegisterGinkgoAlert(routesTestName, "SD-CICD", "Diego Santamaria", "sd-cicd-alerts", "sd-cicd@redhat.com", 4)
 }
 
-var _ = ginkgo.Describe(routesTestName, ginkgo.Ordered, label.E2E, func() {
+var _ = ginkgo.Describe(routesTestName, ginkgo.Ordered, label.HyperShift, label.E2E, func() {
 	var h *helper.H
 	var client *resources.Resources
+	httpClient := &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
+			},
+		},
+	}
 
 	ginkgo.BeforeAll(func() {
 		h = helper.New()
 		client = h.AsUser("")
 	})
 
-	ginkgo.It("exist for console and is operational", label.HyperShift, func(ctx context.Context) {
-		var consoleRoute v1.Route
-		err := client.Get(ctx, consoleLabel, consoleNamespace, &consoleRoute)
-		expect.NoError(err)
-
-		for _, ingress := range consoleRoute.Status.Ingress {
-			response, err := validateRoute(ctx, ingress)
-			Expect(err).NotTo(HaveOccurred(), "failed retrieving %s site", consoleRoute.Name)
-			Expect(response.StatusCode).To(Equal(http.StatusOK))
+	ginkgo.DescribeTable("exists and is operational", func(ctx context.Context, name, namespace string) {
+		if name == "oauth-openshift" && viper.GetBool(config.Hypershift) {
+			ginkgo.Skip("OAuth route is not available on a HyperShift cluster")
```

Comment:
@dustman9000 do we know why there is not an oauth route?

Reply:
```
❯ oc get routes -A
NAMESPACE                             NAME                       HOST/PORT                                                                                         PATH        SERVICES                   PORT    TERMINATION          WILDCARD
open-cluster-management-agent-addon   klusterlet-addon-workmgr   klusterlet-addon-workmgr-open-cluster-management-agent-addon.apps.rw-hyper.l5tf.s3.devshift.org               klusterlet-addon-workmgr   <all>   passthrough          None
openshift-console                     console                    console-openshift-console.apps.rw-hyper.l5tf.s3.devshift.org                                                  console                    https   reencrypt/Redirect   None
openshift-console                     downloads                  downloads-openshift-console.apps.rw-hyper.l5tf.s3.devshift.org                                                downloads                  http    edge/Redirect        None
openshift-ingress-canary              canary                     canary-openshift-ingress-canary.apps.rw-hyper.l5tf.s3.devshift.org                                            ingress-canary             8080    edge/Redirect        None
openshift-monitoring                  alertmanager-main          alertmanager-main-openshift-monitoring.apps.rw-hyper.l5tf.s3.devshift.org                         /api        alertmanager-main          web     reencrypt/Redirect   None
openshift-monitoring                  prometheus-k8s             prometheus-k8s-openshift-monitoring.apps.rw-hyper.l5tf.s3.devshift.org                            /api        prometheus-k8s             web     reencrypt/Redirect   None
openshift-monitoring                  prometheus-k8s-federate    prometheus-k8s-federate-openshift-monitoring.apps.rw-hyper.l5tf.s3.devshift.org                   /federate   prometheus-k8s             web     reencrypt/Redirect   None
openshift-monitoring                  thanos-querier             thanos-querier-openshift-monitoring.apps.rw-hyper.l5tf.s3.devshift.org                            /api        thanos-querier             web     reencrypt/Redirect   None
```

Reply:
AFAIK, the oauth operator will only be running on the control plane nodes for HCP clusters, that's why you are not seeing it here.